app/views/assistance/_assistance.html.haml

```diff
@@ -1,23 +1,32 @@
 .ui.segment.assistance
   %h4.ui.header= assistance.title
   %p= assistance.institution.name
-  - if assistance.expert
+  - assistance.experts.each do |expert|
     %h5= t('.contact')
-    - contact = assistance.expert
+    - contact = expert
     %p
       = contact.full_name
       %br
       = contact.role
     - expert_email = contact.email
     - mailto_text = t('.contact_by_email')
-  - else
+
+    - if visit.visitee and expert_email
+      - email_body = render partial: 'visits/email_to_expert', locals: { visit_date: visit.happened_at_localized, company_name: visit.company_name, advisor_user: current_user, company_user: visit.visitee, need: question.label, offer: assistance.title, email_specific_sentence: assistance.email_specific_sentence, expert_institution: assistance.institution.name }
+      - email_body = strip_tags email_body
+      - mailto_data = { mailto_log_path: mailto_logs_path(mailto_log: {question_id: question.id, visit_id: visit.id, assistance_id: assistance.id}), mailto_logged: false }
+      = mail_to expert_email, mailto_text, bcc: ENV['APPLICATION_EMAIL'], body: email_body, subject: "#{t('app_name')} - #{t('.company_needs_you', company_name: visit.company_name )}", id: "mailto-expert-#{assistance.id}-button", class: 'ui button green', target: :_blank, data: mailto_data
+    - else
+      = link_to mailto_text, edit_visitee_visit_path(visit, diagnosis_id: diagnosis_id), class: 'ui button green'
+
+  - if assistance.experts.size == 0 and assistance.institution.email
     - expert_email = assistance.institution.email
     - mailto_text = t('.contact_institution_by_email')
+    - if visit.visitee and expert_email
+      - email_body = render partial: 'visits/email_to_expert', locals: { visit_date: visit.happened_at_localized, company_name: visit.company_name, advisor_user: current_user, company_user: visit.visitee, need: question.label, offer: assistance.title, email_specific_sentence: assistance.email_specific_sentence, expert_institution: assistance.institution.name }
+      - email_body = strip_tags email_body
+      - mailto_data = { mailto_log_path: mailto_logs_path(mailto_log: {question_id: question.id, visit_id: visit.id, assistance_id: assistance.id}), mailto_logged: false }
+      = mail_to expert_email, mailto_text, bcc: ENV['APPLICATION_EMAIL'], body: email_body, subject: "#{t('app_name')} - #{t('.company_needs_you', company_name: visit.company_name )}", id: "mailto-expert-#{assistance.id}-button", class: 'ui button green', target: :_blank, data: mailto_data
+    - else
+      = link_to mailto_text, edit_visitee_visit_path(visit, diagnosis_id: diagnosis_id), class: 'ui button green'
```

Comment:
We should find a way to factorize mailto logic. We could do it in a mob programming.